src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+\paragraph{Point spread function:}
+\gammapy allows user to treat different kinds of PSFs,
+in particular, parametric multi-dimensional gaussians (\code{EnergyDependentMultiGaussPSF})
+or King profile functions (\code{PSFKing})
+one. The \code{EnergyDependentMultiGaussPSF} class is able to handle up to three
+gaussians, defined in terms of amplitudes and sigma given for each true energy
+and offset angle bin. Similarly, \code{PSFKing takes} into account the gamma and
+sigma parameters as defined here. The \code{ParametricPSF} allows to create a
+PSF with a representation different from gaussian(s) or King profile(s).
+Finally, the user can take advantage from the \code{PSFMap} class, which creates
+a multi-dimensional map of the PSF in WCS coordinates. At each position, a PSF
+kernel map (\code{PSFKernel}) provides a PSF as a function of the true energy.
+The creation of PSF kernel maps, where the PSF is defined for each
+sky-position, is also given to the user. The latter two can speed up analyses.
+
+\paragraph{Energy dispersion:}
+Edisp, in \iact, is generally given in terms of the so
+called migration parameter ($\mu$), which is defined as the ratio between the
+reconstructed energy and the true energy. This ratio should be as close as one
+and its dispersion can assume the shape of a gaussian (or even more complex
+distributions). Migration parameter is given at each offset angle and
+reconstructed energy. The main sub-classes are \code{EnergyDispersion2D},
+designed to interpret Edisp, \code{EDispKernelMap}, which builds an Edisp kernel
+map, i.e., a 4-dimensional WCS map where at each sky-position is associated an
+Edisp kernel. The latter is a representation of the Edisp as a function of the
+true energy only thanks to the sub-class \code{EDispKernel}.
+
+\paragraph{Background model:}
+The BKG can be represented in \gammapy as either 1) a 2D map (\code{Background2D})
+of count rate normalised per steradians and energy at different
+energies and offset-angles or 2) as rate per steradians and energy, as a
+function of reconstructed energy and detector coordinates (\code{Background3D}).
+In the former, the background is expected to follow a radially symmetric shape,
+while in the latter, it can be more complex.
+
+
+\subsection{gammapy.maps}
+\label{ssec:gammapy-maps}
+The \code{gammapy.maps} sub-package provides classes that represent data
+structures associated with a set of coordinates or a region on a sphere. In
+addition it allows to handle an arbitrary number of non-spatial data
+dimensions, such as time or energy. It is organized around three types of
+structures: geometries, sky-maps and map axes, which inherit from the base
+classes \code{Geom}, \code{Map} and \code{MapAxis} respectively.
+
+The geometry object defines the pixelization scheme and map boundaries. It also
+provides methods to transform between sky and pixel coordinates. Maps consist
+of a geometry instance together with a Numpy data array containing the corresponding
```

Comment:
First mention of Numpy! need a reference here...

The publication scheme of Numpy is here: [citing Numpy](https://numpy.org/citing-numpy/)